### ISSUE TITLE
Training loss and validation loss 

### ISSUE THREAD

**Issue by abhigoku10** [Mandylove1993/complex-yolo#3]
@Mandylove1993 Hi thanks for the wonderful code . In you current code have you considered the validation loss of your training
